scripts/trlite

```diff
@@ -446,7 +446,7 @@ if [ "$RUN" == "all" -o "$RUN" == "4" ]; then
     try_command "ashell" make $VERBOSE ashell ROM=250
 
     # build ide version
-    try_command "ide" make $VERBOSE ide ROM=250
+    try_command "ide" make $VERBOSE ide ROM=251
```

Comment:
I merged your patch already so this will probably get taken care of in rebase. I'll try that in minute.